### PR TITLE
First post in Getkw series

### PR DESCRIPTION
**Pull request by robertodr**
Details status and requirements.

### CODE REVIEW

File: _posts/2018-09-14-getkw-01.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+### Strengths
+
+a. Invalid inputs are blocked before CPU cycles are consumed.
+b. No regexes are used, rather the input is treated as a formal grammar with its
+   [Backus-Naur form].
+
+### Weaknesses
+
+a. Getkw is:
+   - A Python module. This uses `pyparsing.py` to define the grammar for all
```
Comment (bast) [inline _posts/2018-09-14-getkw-01.md]: But why is it a weakness that getkw is a python module and uses `pyparsing.py`? Not questioning this statement but the justification may not be clear to the reader - it is not clear to me.

**Review by bast — APPROVED** [review]
I have a couple of minor modifications/addition to this post but perhaps I rather communicate them as PR after this one is in?

File: _posts/2018-09-14-getkw-01.md (hunk 1 above)
Comment (robertodr) [reply _posts/2018-09-14-getkw-01.md]: It's the combination of Python/compiled languages that is a weakness, IMO. Though I agree that it is not very clear now, perhaps

**Comment by robertodr** [comment]
Make changes directly here and add yourself as coauthor of the post.

**Comment by robertodr** [comment]
Thanks for your changes! You can go ahead and merge. My suggestion for this series of posts is to leave them in the "halfbaked" section for a while (how long is to be decided) so others can comment and make changes through PRs too.

**Comment by bast** [comment]
I agree.

**Pull request merged**